dashboard/src/components/AppView/AccessURLTable/AccessURLTable.tsx

```diff
@@ -77,18 +77,34 @@ function flattenIngresses(ingresses: Array<IKubeItem<IResource | IK8sList<IResou
 }
 
 function getAnchors(URLs: string[]) {
-  return URLs.map(URL => (
+  return URLs.map(URL => getAnchor(URL));
+}
+
+function getAnchor(URL: string) {
+  return (
     <div className="margin-b-sm">
       <a href={URL} target="_blank" rel="noopener noreferrer" key={URL}>
```

Comment:
running the tests for this, I noticed several warnings like `Warning: Each child in a list should have a unique "key" prop.`. For this function, the `key={URL}` needs to be in the `div`. Note that any component returned in an array should have the `key` property so you need to update `getSpan` and `getUnknown`.